### PR TITLE
Automation-Changes-A

### PR DESCRIPTION
**Pull request by RealRadOne**

### CODE REVIEW

File: src/client.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    
+def getTags(tagsText):
+    tagsText = tagsText.replace('\n','')
+    tagsText.strip()
+    tagList = tagsText.split(',')
+    while( len(tagList) < 4 ):
+        tagList.append('')
+    return tagList[:4]
+
+outCSV = open("out.csv","w")
```
Comment (AbstractXan) [inline src/client.py:34]: This should be renamed to `data.csv` since LINK_PUMP reads looks for `data.csv`

File: src/build.sh [hunk 2]
```diff
@@ -1,3 +1,4 @@
+python3 client.py
```
Comment (AbstractXan) [inline src/build.sh:1]: Might wanna rename to something better

File: src/client.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
+from telethon import TelegramClient, sync
+import pandas as pd
+import json 
+
+api_id = 2899
+api_hash = '36722c72256a24c1225de00eb6a1ca74'
```
Comment (AbstractXan) [inline src/client.py:6]: You sure you wanna put this out to public ?

**Review by AbstractXan — CHANGES_REQUESTED** [review]
Works great. We might have to edit Mizi's hard coded config to get all our data appropriately.

File: src/client.py (hunk 3 above)
Comment (RealRadOne) [reply src/client.py:6]: Yes,they are telegram public api keys

**Review by AbstractXan — CHANGES_REQUESTED** [review]
We don't wanna lose `src/data.csv`

**Comment by RealRadOne** [comment]
> We don't wanna lose `src/data.csv`

**Review by AbstractXan — APPROVED** [review]

**Review by AbstractXan — APPROVED** [review]

**Comment by AbstractXan** [comment]
The changes seem correct. However the Netlify builds are not working right. Merging this PR and fixing this on master.

File: src/extparse.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
+from telethon import TelegramClient, sync
+import pandas as pd
+import json 
+
+api_id = 2899
+api_hash = '36722c72256a24c1225de00eb6a1ca74'
```
Comment (siddish-reddy) [inline src/extparse.py:6]: let's shift this to env variable if it has read and also write authorization

Comment (AbstractXan) [reply src/extparse.py:6]: Yeah I asked @RealRadOne if she was okay with this -- she said yes.

**Pull request merged**